### DISCUSSION TITLE
Some layer changes on dashboard QFieldCloud are not applied and I don't understand

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#2466] [Q&A]
On the QFieldCloud dashboard I see that some changes from a collaborator are 'not applied', others of his changes are. I don't understand what is wrong. When I open the details (print-screen) it says Action: "patch" (which I don't understand neither). Could someone explain me? 

Next, I have the choice: 'schedule to apply' or 'Permanently Ignore it'. With the details of 'raw changed' I see what it's about, but I can't really decide whether to keep it or not because I don't see the change in the project itself.

Thank you.

![afbeelding](https://user-images.githubusercontent.com/57868975/151712442-fdb0f169-b9d7-4a94-986a-0a229586ad12.png)
---
![afbeelding](https://user-images.githubusercontent.com/57868975/151712587-14000052-015f-4010-aefe-d6b3f7631995.png)

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@BoswachterMarc hey, please check the latest version of QFieldCloud. Since this page design have significantly change, please open a new discussion Q/A if things are still unclear.
